basics/install.md

```diff
@@ -124,7 +124,10 @@ Once Graal is installed, you can move ahead to check whether it's installed corr
 (gl) $ graal --help
 ```
 
-**Note:** CoLic backend requires executable path of tools which it leverages on ( [NOMOS](https://github.com/fossology/fossology/tree/master/src/nomos) & [SCANCODE](https://github.com/nexB/scancode-toolkit) ). You can install them by following the procedure mentioned [here](https://github.com/chaoss/grimoirelab-graal#how-to-installcreate-the-executables).
+**Note:** 
+1. Graal requires installation of some executables for its execution. More information on installing the executables can be found [here](https://github.com/chaoss/grimoirelab-graal#how-to-installcreate-the-executables)
```

Comment:
At the end of the first sentence, could we mention cloc, something like: (e.g., cloc) should work. In this way we can address this issue: https://github.com/chaoss/grimoirelab-tutorial/issues/97. wdyt?

Reply:
@valeriocos I didn't add it as I thought, there could be more additions(executable, probably) in the future.
But, Agree with you. We could add it as shown (eg.)